network-store-integration-test/src/test/java/com/powsybl/network/store/integration/NetworkStoreIT.java

```diff
@@ -602,6 +603,7 @@ public void threeWindingsTransformerTest() {
     }
 
     @Test
+    @Ignore
```

Comment:
Cann you just add a text for the @Ignore annotation, to explain why this test is temporarly ignored ?

Reply:
Ok. And I opened an issue